CONFIG.md

```diff
@@ -275,6 +275,55 @@ directly to
 [grunt-contrib-compass](https://github.com/gruntjs/grunt-contrib-compass)
 for this theme.
 
+### Proxying Grunt Tasks
+
+Many themes come with their own Grunt integration. In those cases you can
+configure Grunt-Drupal-Tasks for awareness of that and remove the theme
+from most handling.
+
+In order to exclude your theme from handling, specify the `proxy` property in
```

Comment:
On further consideration, I'm subscribing to the generic `proxy` approach. It seems like it would not be especially useful to differentiate if GDT is proxying to a Grunt- vs. Gulp- vs. shell-based theme script, since we'd invoke any of these scripts through a grunt-shell task. Given this, I'm learning towards including the command in the config structure (at least for the more advanced approach).

What do you mean by losing "the ability to pass along any other flags"? Do you mean flags that GDT would automatically apply or flags that could be configured in Gruntconfig.json? I'm not sure how I feel about automatically passing options through to the theme script, since it could be possible to include any desired options in the Gruntconfig.json option for the script.

What should the default task mappings be? (Or should we even have a simple default?) Some theme scripts may only have one appropriate mapping: GDT's `compile-theme` (also part of GDT's `default`) triggers a task in the theme. Others may have a corollary to `validate` or `test`, perhaps even `scaffold`.

How does this seem for the advanced configuration where you can map GDT tasks to theme script tasks, specify the command, task/argument, and command line option (optionally):

```
"legionaire": {
  "path": "<%= config.srcPaths.drupal %>/themes/legionaire",
  "proxy": {
    "compile-theme": "grunt compile --force",
    "validate": "grunt lint"
  }
}
```